src/components/SocialIcon/SocialIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from '@/components/SocialIcon/SocialIcon.module.css';
+import facebookIcon from '@/img/social/facebookIcon.svg';
+import instagramIcon from '@/img/social/instagramIcon.svg';
+import linkedinIcon from '@/img/social/linkedinIcon.svg';
+
+const SocialIcon = () => {
+    const facebookLink = 'https://www.facebook.com/david.tereshchenko.77/';
+    const linkedinLink = 'https://www.linkedin.com/in/david-tereshchenko-7bb98b240/';
+    const instagramLink = 'https://www.instagram.com/davidushka_te/';
```

Comment:
If the variables are constants, name it in constant and descriptive way (Ex: INSTAGRAM_URL)